src/components/BetaModal.js

```diff
@@ -1,56 +1,101 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 
-const BetaModal = ({ onConfirm }) => {
-  const fixed = 'fixed top-0 bottom-0 left-0 right-0'
+import { showFeedback } from '../actions/feedback'
+import { hideModal } from '../actions/modal'
 
-  return (
-    <div className={`${fixed} flex flex-column items-center justify-center z2`}>
-      <div className={`${fixed} bg-black-translucent`} />
+
+class BetaModal extends React.Component {
+  componentDidMount() {
+    this.triggerElement = document.activeElement || document.body
```

Comment:
i'm not convinced we need a 'close on escape' (it feels like we want people to read and then click the button)